Mathlib/ModelTheory/Graph.lean

```diff
@@ -33,30 +31,31 @@ open FirstOrder
 
 open Structure
 
-variable {L : Language.{u, v}} {α : Type w} {V : Type w'} {n : ℕ}
+variable {α : Type u} {V : Type v} {n : ℕ}
 
 /-! ### Simple Graphs -/
 
+/-- The type of relations for the language of graphs, consisting of a single binary relation `adj`.
+-/
+inductive graphRel : ℕ → Type
+  | adj : graphRel 2
+  deriving DecidableEq
 
 /-- The language consisting of a single relation representing adjacency. -/
-protected def graph : Language :=
-  Language.mk₂ Empty Empty Empty Empty Unit
+protected def graph : Language := ⟨fun _ => Empty, graphRel⟩
+  deriving IsRelational
 
 /-- The symbol representing the adjacency relation. -/
-def adj : Language.graph.Relations 2 :=
-  Unit.unit
+abbrev adj : Language.graph.Relations 2 := .adj
```

Comment:
Random thought, have you considered
```suggestion
export graphRel (adj)
```
? Not sure it's wanted, just a thought

Reply:
This doesn't accomplish the same thing - for namespaces and a few other technicalities, it is useful to have `adj` be immediately defined as a `Language.graph.Relations 2` rather than the definitionally equal `graphRel 2`.